src/user/approval.js

```diff
@@ -12,6 +13,10 @@ const utils = require('../utils');
 const plugins = require('../plugins');
 
 module.exports = function (User) {
+	new cronJob('0 * * * * *', function () {
```

Comment:
Every minute is definitely too fast, maybe every hour is better

Reply:
Checking every hour might lead to some problems with small wait times - for example having to wait almost 2 hours instead of one.

But I think that's a rare use case...

Perhaps giving the user actual date and time when they'll be automatically approved (`23:00 26.10.2020` instead of `in 1 hour` for example) would be better? Though it would most likely require using moment to format the date for user locale and might be unnecessarily overcomplicated.